ui/src/app/lib/atlasmap-data-mapper/components/mapping/mapping.field.detail.component.ts

```diff
@@ -35,45 +35,54 @@ import { MappingModel, FieldMappingPair, MappedField } from '../../models/mappin
         <ng-template #tolTemplate>
             <div class="fieldDetailTooltip">
                 <label class="parentObjectName" *ngIf="displayParentObject()">
-                    <i [attr.class]="getSourceIconCSSClass()"></i>
-                    {{ getParentObjectName() }}
+                    <i [attr.class]="sourceIconCSSClass"></i>
+                    {{ parentObjectName }}
                 </label>
                 <label>{{ getFieldPath() }}</label>
                 <label *ngIf="displayParentObject() && mappedField.field.type">({{ mappedField.field.type }})</label>
                 <div class="clear"></div>
             </div>
         </ng-template>
 
-        <div class='fieldDetail' style="margin-bottom:5px;" *ngIf="mappedField"
-            [tooltip]="tolTemplate" placement="left">
+        <div class='fieldDetail' style="margin-bottom:5px;" *ngIf="mappedField">
             <label class="parentObjectName" *ngIf="displayParentObject()">
-                <i [attr.class]="getSourceIconCSSClass()"></i>
-                {{ getParentObjectName() }}
+                <i [attr.class]="sourceIconCSSClass"></i>
+                {{ parentObjectName }}
+                <span class="pficon pficon-info" [tooltip]="tolTemplate" placement="left"></span>
             </label>
             <div style="width:100%;">
-                <input type="text" id="input-{{isSource?'source':'target'}}-{{mappedField.field.getFieldLabel(false)}}"
+                <input type="text" id="{{inputId}}"
```

Comment:
just out of curiosity, we are we binding interpolated values to an `id` attribute? I'm just curious...

Reply:
@deeleman , I almost overlooked before your comment - so the id is required to capture input fields in the HTML for the e2e test framework. @mmelko , isn't this change breaking what we want for e2e framework?

Reply:
@deeleman I don't know why we do this. It's existing code. It looks odd to me. @igarashitm I didn't change any logic related to how we build the id value with the goal of not breaking any e2e tests, in case we depend on it.

Reply:
Now it makes sense. Thanks for the explanation guys! ;)

Reply:
@deeleman as it was mentioned before, It's used in syndesis e2e tests. @akieling I agree not the nicest but it works. Thanks for keeping it ;) We might want to change logic of building id in the future though.